src/lib/data.ts

```diff
@@ -170,3 +204,29 @@ try {
 } catch (e) {
     console.error(e);
 }
+
+// update local data store from persisted values (which may have been modified by other devices)
+
+try {
+    remote_storage.get('persisted-local-storage')
```

Comment:
If I'm not mistaken, having this `get` here and the one up above in `set` could result in a bit of a race condition. I'm thinking we should probably have a single place where we manage that synchronization, what do you think?

Reply:
Oh, are you pointing out that code can start setting stuff before this initialisation even finishes?

Right - I hadn't considered that case.   I'll tidy that up.